python/doc/examples/probabilistic_modeling/overview_univariate_distributions.ipynb

```diff
@@ -0,0 +1,602 @@
XgLfeAgBIHh4w8uoVuQsvL2D+fGDrVuugpMeOAZGRYl5D3jIkAsACi+juLFoEXLgAADjzl78AbdrIHBCRkz3xBJCXJzq8A8CNG+Ip2sceA06fljc2ojqABRbRnbp0CXjzTQCApFbjyPDhMgdEJJPQUGDvXmDCBGvbrl1Ap07A+vW8mkVujQUW0Z2aO9fSqVd6+mmUBgfLHBCRjLRa4N13gc8/F320AODKFWDkSGDoUMuVXiJ3o9gCa+nSpWjRogW0Wi0iIyORm5tb47YrVqxAz5490bBhQzRs2BDR0dG/uz1RjU6fBszT4mi1ME6bJm88RHXFY48BP/4IjBhhbfv4YzH1ztat8sVFJBNFFljp6elITExEcnIy8vLyEBYWhtjYWJw/f77a7bOzszF8+HDs3r0bOTk5CAkJQUxMDM6cOePkyEnxXn8dMI97NXGidVJcIgIaNhS3BjdvBu6/X7SdPw8MGSKWoiJ54yNyIkUWWAsXLsT48eMxZswYtG/fHmlpafD19cWqGiYj3bBhA55//nmEh4ejXbt2eP/992EymZCVleXkyEnRDh+2Tnjr7w+88oq88RDVVUOHAj/9BAwaZG3buhV48EFg3jzAPLUUkQtTXIFVXl6OAwcOIDo62tKmUqkQHR2NnJycWn1GWVkZKioq0KhRI0eFSa5GkoDJk60T3U6ZYv2GTkRVBQSIomrjRkCnE21lZcCrr4onD/fskTc+IgfzlDuAO3Xx4kUYjUYE3DYlSUBAAI4cOVKrz3jllVcQFBRkU6Td7ubNm7hpvhUEwPBbp+aKigpUVFTcReTOZ45TKfHWZR7btsFz1y4AgNS8OSpffBG45Vxgjh2HOXY8h+Z46FCgXz+oZs6EatkyeJhMwKFDQO/eMP397zCmpLjFFFPufh6743ErrsC6V3PnzsWHH36I7OxsaLXaGrdLSUnBa6+9VqV9165d8PX1dWSIdpeZmSl3CIqmunkTj06caPll+f6vf8W53btttmGOHY85djyH5rhfP9QPDUVYWhoaHjsGAFCtXw/TRx/h2JAhODFoEIw+Po77+XWEu57HZWVlcofgdB6SpKyBSsrLy+Hr64uPPvoIgwcPtrSPHj0aV65cwbZt22rcd8GCBXjjjTfwxRdfoFu3br/7c6q7ghUSEoKLFy/C39//3g/ECSoqKpCZmYl+/frBy8tL7nAUSzV7NtS/FdumRx+F8fPPAQ8PAMyxMzDHjufUHBuNUK1aBdW0aZappgBA0ulgTEqC9PTTgKfrffd39/PYYDCgcePGuHr1qmL+D71XijuLNRoNunbtiqysLEuBZe6wnpCQUON+8+fPx+zZs5GRkfGHxRUAeHt7w9vbu0q7l5eX4n45lBhznVFUJKYFAQBPT6hSU6HSaKpsxhw7HnPseE7JsZeXGPF96FDxVO7y5UBlJTz0eng+/zyweLHoCD9woOWLjCtx1/PYHY9ZcZ3cASAxMRErVqzA2rVrcfjwYUyYMAGlpaUYM2YMAGDUqFGYOnWqZft58+ZhxowZWLVqFVq0aAG9Xg+9Xo9r167JdQikBJIk/iO4fl2sT5wItG8vb0xErqJJE2DJEvG04f/8j7X9yBExDc8jjwC7d3M0eFIsRRZY8fHxWLBgAZKSkhAeHo78/Hzs3LnT0vG9qKgI586ds2y/bNkylJeXY+jQoQgMDLQsCxYskOsQSAnWrwe2bxevdTogOVneeIhcUZs2wEcfAd9+C3Tvbm3/5hvg0UeBnj2BjAwWWqQ4irtFaJaQkFDjLcHs7Gyb9ZMnTzo+IHIt584BkyZZ15cvB+rXly8eIlfXvbsoqrZuFUM5/Pe/ov3bb8Uo8RERwIwZQFycS946JNejyCtYRA4lSWLy2suXxfrf/mY7YCIROYaHhxjx/dAhYMMG21vyubmiX1bXrmKk+MpK+eIkqgUWWES3+/BDwPw0apMmotMtETmPWi2+2BQUiGLqoYes7x08CDz1FNCqFbBggfWLEFEdwwKL6FYnT4qO7WbvvssR24nkolKJpw0PHhS3Drt2tb5XVCRmVAgJARISrLcUieoIFlhEZuXlQHw8YB6bJz7e9ukmIpKHSiWeLPz+eyAzExgwwPpeaSmwdCnQtq3on/Wf//D2IdUJLLCIzKZOFf08ACA0VHRsJ6K6w8MDiI4WRdTRo+LKlZ+f9f0dO0R/yebNgenTgZ9/li9WcnsssIgA8Qd74ULx2ssLSE/nU4NEdVmbNkBqKnD6tOiL1ayZ9b2zZ4HZs0U/reho8ft8y8wcRM7AAovo2DFg9Gjr+oIFQC1G+yeiOqBBA+Cf/wROnAA++wwYPFh0kjfLygL++lcgMBAYP14MXmo0yhcvuQ0WWOTefv1V9NswP4k0ZIgYsZ2IlMXTU/wuf/IJcOoUkJIirmCZXb4MvP++GLy0WTNRlB04wAFMyWFYYJH7Ki8HnnxSXMECgA4dgNWrOYghkdIFBloHK929G/j73237ap09K7oEdOsmOsdPmyY60JtM8sVMLocFFrknSQL+93+BPXvEepMm4vYC+10RuQ6VCujdG1i3DiguFmPcDRok+lmaHTsGzJkjRopv1gx44QXxpGJFhWxhk2tggUXuR5KAf/0LWLtWrGu1YmDRFi1kDYuIHMjPTwy9sm0boNcD770H9Olje8X6zBkx9l1MDPCnPwEjRog5Sc+fly9uUiwWWOR+kpOBuXOt62vXAn/+s3zxEJFzNWokOrx/+aWYd3TFCtF/S6OxbnP1KrBxIzByJBAQIAY5nTYN+PprXt2iWmGBRe5l1iyxmL37rph2g4jcU0AAMG6c6CJw8SKwaZOYpuf27gJ5eeJW4iOPAI0bi4FPFy0C8vPZd4uq5Sl3AEROIUnAa6+JxWzxYjGpMxERANSrBwwbJpbyciAnB9i5E8jIENP1mBkMwKefigUAGjYEevUS/b169wY6drQdKoLcEgsscn0VFcBzzwGrVlnb3nqLwzEQUc00GlE09eolhnzQ64Fdu0T
```

Comment:
Another Github suggestion failure:
In order to check the previous distribution, we compare it with the LogNormal distribution.

["Compare to" vs "compare with"](https://www.dailywritingtips.com/compared-to-or-compared-with/)

Reply:
True. Fixed.